config.map

```diff
@@ -1,8 +1,7 @@
 SCRAM_ARCH=el8_amd64_gcc11;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_13_1_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_13_1_X;DOCKER_IMG=cmssw/el8;BUILD_PATCH_RELEASE=1;ADDITIONAL_TESTS=HLT,static-checks,fwlite,igprof-mp,igprof-pp,iwyu,lizard,flawfinder,check-headers,cfipython,invalid-includes,profiling,crab;PR_TESTS=1;PROD_ARCH=1;PRS_TEST_CLANG=1;REQUIRED_TEST=true;SPACK=1;
 SCRAM_ARCH=slc7_amd64_gcc11;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_13_1_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_13_1_X;DOCKER_IMG=cmssw/cc7;BUILD_PATCH_RELEASE=1;
 SCRAM_ARCH=el8_amd64_gcc11;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_13_1_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_13_1_DBG_X;DOCKER_IMG=cmssw/el8;BUILD_HOUR=23;BUILD_DAY=4;
-SCRAM_ARCH=el8_aarch64_gcc11;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_13_1_X/lto;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_13_1_LTO_X;DOCKER_IMG=cmssw/el8;
-SCRAM_ARCH=el8_amd64_gcc11;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_13_1_X/lto;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_13_1_LTO_X;DOCKER_IMG=cmssw/el8;PR_TESTS=1;
+SCRAM_ARCH=el8_amd64_gcc11;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_13_1_X/nonlto;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_13_1_NONLTO_X;DOCKER_IMG=cmssw/el8;PR_TESTS=1;
```

Comment:
@aandvalenzuela , I would suggest to keep the LTO ibs in the config file so that previously built LTO Ibs do not disappear from the IB page. We can remove these in couple of weeks ( when LTO ib will be deleted from cvmfs)

```suggestion
SCRAM_ARCH=el8_amd64_gcc11;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_13_1_X/nonlto;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_13_1_NONLTO_X;DOCKER_IMG=cmssw/el8;PR_TESTS=1;
SCRAM_ARCH=el8_aarch64_gcc11;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_13_1_X/lto;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_13_1_LTO_X;DOCKER_IMG=cmssw/el8;DISABLED=1;IB_WEB_PAGE=1;
SCRAM_ARCH=el8_amd64_gcc11;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_13_1_X/lto;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_13_1_LTO_X;DOCKER_IMG=cmssw/el8;PR_TESTS=1;DISABLED=1;IB_WEB_PAGE=1;
```